cmake/SuiteSparse-config-install.cmake.in

```diff
@@ -0,0 +1,36 @@
+# Compute locations from <prefix>/@{LIBRARY_DIR@/cmake/suitesparse-<v>/<self>.cmake
+get_filename_component(_SuiteSparse_SELF_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_SELF_DIR}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_PREFIX}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_PREFIX}" PATH)
+
+# Load the LAPACK package with which we were built.
+find_package(LAPACK CONFIG)
```

Comment:
yes, those code blocks are used only if no `lapack-config.cmake` is found AND when on Windows.
The `else()` path of `if(LAPACK_FOUND)` is not indented. Maybe this should be changed in another PR